charts/greptimedb-operator/values.yaml

```diff
@@ -27,10 +27,10 @@ replicas: 1
 resources:
```

Comment:
You can leave this modification to the new PR with the new chart version. We should keep one PR one thing.

Reply:
done